lib/ansible/plugins/connection/ssh.py

```diff
@@ -296,7 +296,9 @@
             version_added: '2.7'
       sftp_batch_mode:
         default: true
-        description: 'TODO: write it'
+        description: '
+        - If set to C(true), Ansible will use SFTP batch mode for transfers. This can optimize file transfers, particularly for larger files.
+        - Set to C(false) to disable batch mode and send files one at a time.'
```

Comment:
```suggestion
        description:
        - If set to C(true), Ansible will use SFTP batch mode for transfers. This can optimize file transfers, particularly for larger files.
        - Set to C(false) to disable batch mode and send files one at a time.
```